src/RecordManager/Base/Record/Marc.php

```diff
@@ -634,7 +635,7 @@ public function getLinkingIDs()
             $results = [...$results, ...$cns];
         }
 
-        return $results;
+        return array_unique(array_filter($results));
```

Comment:
Oops, forgot that this is actually used for the database too. So needs array_values as well.
```suggestion
        return array_values(array_unique(array_filter($results)));
```